frontend/src/components/chats/components/current-chat/components/messages-list/components/date-separator/date-separator.tsx

```diff
@@ -0,0 +1,22 @@
+import { FC } from 'common/types/types';
+
+import styles from './styles.module.scss';
+
+type Props = {
+  postTime: string;
+};
+
+const DateSeparator: FC<Props> = ({ postTime }) => {
+  return (
+    <div className={styles.dateSeparator}>
+      {`${new Date(postTime).getDate()} ${new Date(postTime).toLocaleString(
```

Comment:
use `getFormattedDate` helper

Reply:
And for today's date I think it would be better to have `Today` label